tools/run_migration

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+# Copyright (c) 2019 SUSE Software Solutions Germany GmbH.  All rights reserved.
+# 
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>
+#
+# Enable shell globbing
+set +o noglob
+
+logfile=/var/log/migration_startup.log
+
+function setup_logging {
+    rm -f "${logfile}"
+    exec 2>>"${logfile}"
+    set -x
+}
+
+function cleanup {
+    # shellcheck disable=SC2181
+    if [ ! $? = 0 ];then
+        echo "Migration run failed: see ${logfile} for details"
+    fi
+    umount /mnt &>/dev/null
+}
+
+function get_migration_image {
+    # """
+    # Search migration ISO file
```

Comment:
"Search for ..."

Reply:
I should take part in an English class. We had that at suse years before and I think a refresh would be good :) Thanks